model/user/user.go

```diff
@@ -67,9 +67,9 @@ type UserSettings struct {
 }
 
 type UseSpruceOptions struct {
-	PatchPage           bool `json:"patch_page" bson:"patch_page"`
-	SpruceV1            bool `json:"spruce_v1" bson:"spruce_v1"`
-	HasUsedSpruceBefore bool `json:"has_used_spruce_before" bson:"has_used_spruce_before"`
+	PatchPage           bool `json:"patch_page" bson:"patch_page"`                         // deprecated - do not use
```

Comment:
I think it makes sense to remove the PatchPage field completely from Evergreen. We can do that in this PR or I can make a ticket for it